meta-lmp-base/classes/lmp-staging.bbclass

```diff
@@ -15,6 +15,10 @@ python __anonymous() {
 
     if bb.data.inherits_class('go-mod', d):
         d.appendVarFlag('do_compile', 'network', '1')
+
+    pn = d.getVar('PN')
+    if pn in ["clang", "rust"]:
```

Comment:
Will this catch native runs as well? Or is that already safe with the normal dependency chain locks?

Reply:
no, this only lock the target ones. but yes, the dependency chain already block the native ones.

`clang` needs `clang-cross-x86_64`  that pulls `clang-native`
```
bitbake-getvar -r clang DEPENDS
DEPENDS="pkgconfig-native cmake-native  binutils zlib zstd libffi libxml2 libxml2-native ninja-native swig-native python3-native  clang-cross-x86_64 python3 compiler-rt libcxx libedit libedit-native ncurses ninja-native"
```

`rust` needs `rust-native` and `clang-native`

```
bitbake-getvar -r rust DEPENDS
DEPENDS=" clang-cross-x86_64 virtual/libc  compiler-rt libcxx  file-native python3-native rust-llvm (=1.63.0) python3-native   rust-native libstd-rs clang-native compiler-rt-native libcxx-native"
```

So the order without this patch is:

1. `clang-native`
2. `rust-native`
3. `clang` and `rust` that can run at same time